### PR TITLE
Fallback to previous versions when failing to create translator

### PR DESCRIPTION
**Pull request by elfinhe**
Fallback to previous versions when failing to create translator and reversed translator, in case we may forget to update the version specific files (e.g., TranslateConfig and ReverseTranslators Map) when releasing a new minor version.

### CODE REVIEW

File: pkg/translate/translate.go [hunk 1]
```diff
@@ -109,10 +109,22 @@ type Translation struct {
...
 	f := "translateConfig/translateConfig-" + v + ".yaml"
 	b, err := vfs.ReadFile(f)
 	if err != nil {
+		if fallbackNum > 0 && minorVersion.Minor > 0 {
+			major := minorVersion.Major
+			minor := minorVersion.Minor - 1
+			previous := version.NewMinorVersion(major, minor)
+			log.Warnf("could not read translateConfig file for version %s, fallback to version %s",
+				minorVersion, previous)
+			return newTranslator(previous, fallbackNum-1)
```
Comment (richardwxn) [inline pkg/translate/translate.go]: I don't quite get the usage of fallback num here, so looks like it is hardcoded to 1 at the exported method, then in this recursion, next call would always have fallbacknum equals 0 so it has only two states. If that's the case why we still need to call newTranslator function again

Comment (elfinhe) [reply pkg/translate/translate.go]: I can rename the '1' to a constant `maxFallbackNum`. Sorry, I do not get your idea. What's your suggestion?

Comment (elfinhe) [reply pkg/translate/translate.go]: Fixed.

**Review by richardwxn — APPROVED** [review]

**Pull request merged**